ci/s3-uploader.sh

```
@@ -71,58 +74,74 @@ bucket=`printf $3 | awk 'BEGIN{FS="@"}{print $1}'`                       # bucke
+# Default: `-f` makes curl exit non-zero on HTTP >=400 (so callers detect
+# failures) but it also SUPPRESSES the response body. Set S3_UPLOADER_DEBUG=1
+# to drop -f and print the server's error XML (for debugging 403s etc).
+if [ -n "${S3_UPLOADER_DEBUG:-}" ]; then
+    CURL_FAIL_FLAGS="-S"
+else
+    CURL_FAIL_FLAGS="-f"
+fi
 curl -vvv                       \
-    -# -kf                      \
+    -# -k $CURL_FAIL_FLAGS      \
     -F key=$targfile            \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)